pom.xml

```diff
@@ -182,6 +189,26 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>enforce-java</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>[1.6.0,)</version>
```

Comment:
Is this a mismatching bracket?

Reply:
Nvm, I think this is correct.

Reply:
Yes, this is correct based on this: https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html